macros.in

```diff
@@ -1063,6 +1063,10 @@ package or when debugging this package.\
 #
 %ix86   i386 i486 i586 i686 pentium3 pentium4 athlon geode
 
+#------------------------------------------------------------------------------
+# arch macro for all supported x86_64 processors
+%x86_64	x86_64 x86_64_v2 x86_64_v3 x86_64_v4 amd64 em64t
```

Comment:
I wonder if that should be called "x86_64v2" instead. we do not have any other architectures with underscored suffixes as far as I can see

Reply:
Is there a reason we can't use `-` there?

Reply:
I'm not aware of a reason.

Reply:
`-` is used as separator between nevra components in various places, I didn't want to cause any issues in that regard.

The canonical name of those levels is `x86-64-vX`, so I just continued the `s/-/_/` trend that Linux and RPM started. I don't really care myself though.

Reply:
Yeah, mixing - and _ in the same name would only make it so much more painful to type.

Reply:
I've slept on this and thought about it, and I think I like @dirkmueller's original suggestion of just dropping the `-` instead of converting it to an underscore. It's slightly more readable and less confusing. Can you do that, please?